doc/cla/corporate/brainbeanapps.md

```diff
@@ -0,0 +1,15 @@
+Estonia, 23 OCT 2018
+
+Brainbean Apps OU agrees to the terms of the Odoo Corporate Contributor License
+Agreement v1.0.
+
+I declare that I am authorized and able to make this agreement and sign this
+declaration.
+
+Signed,
+
+Oleksii Pelykh <alexey@brainbeanapps.com> https://github.com/alexey-pelykh
+
+List of contributors:
+
+Alexey Pelykh <alexey.pelykh@gmail.com> https://github.com/alexey-pelykh
```

Comment:
shouldn't it be your corporate email address?

Reply:
Well, both corporate & personal emails are bound to same github account. Don't really think it's important, yet if that email is not verified by tools, I think it would be better to specify corporate one. What would be your suggestion?

Reply:
yes, you can configure multiple emails per account (this is what I do, using the same account for both professional and personal projects). However, the CLA bot will checked based on the email used to make the commits. It appears you are making the commits using your gmail account. I advise you to set the right email (`git config user.email`) for each repository to commit with the right email.

Reply:
Awesome, fixed